training/src/app/chuck-norris-main/chuck-norris-main.component.css

```diff
@@ -0,0 +1,117 @@
+
+.top-bar > #top-bar-img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 80%;
+    height: 60%;
+}
+
+.buttons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.buttons > #toggle-buttons {
+    border-radius: 4px 4px;
+    margin-top: 3%;
+    margin-bottom: -5%;
+    height: 30px;
+    border: 1px solid #F05924;
+}
+
+.buttons-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 0.1fr;
+    grid-gap: 20px 50px;
+    background-color: #FEF2ED;
+    padding: 2% 3%;
+    border-radius: 0.4em;
+    margin: 8% 20% 3%;
+}
+
+.buttons-container > #input-style {
+    border: 0 #ffffff;
+    border-radius: 0.4em;
+    width: 90%;
+}
+
+.buttons-container > button {
+    width: 70%;
+    color: white;
+}
+
+.mat-select {
+    background-color: #ffffff;
+    border-radius: 5px;
+    width: 90%;
+}
+
+.white {
+    background-color: #ffffff;
+    color: #F05924;
+    width: 80px;
+    height: 50px;
+    margin-top: -4px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 16px;
+    font-family: 'Roboto', sans-serif;
```

Comment:
line 57,58,61
there is no need to write it again in several classes in this file.
if this is the default for all, u can write it the top class,
for example u can put it in ".buttons" class, then u dont need it in white / orange classes.
try to avoid duplications